S/Spot_julia/build_tarballs.jl

```diff
@@ -0,0 +1,105 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Spot_julia"
+version = v"2.9.7"
+julia_version = v"1.6.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://www.lrde.epita.fr/dload/spot/spot-2.9.7.tar.gz","1eea67e3446cdbbbb705ee6e26fd869020cdb7d82c563fead9cb4394b9baa04c"),
+    GitSource("https://github.com/MaximeBouton/spot_julia.git", "9b59bebb96a973ba078f482128e3aeac1fe3cc38")
+    ]
+    
+    # Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/spot-2.9.7/
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-python
+make -j${nproc}
+make install
+
+# build with cmake 
+cd $WORKSPACE/srcdir/spot_julia/spot_julia
+
+if [[ $target == *"mingw"* ]]; then
+  cp -L ${prefix}/bin/libspot-0.dll ${prefix}/lib/
+  cp -L ${prefix}/bin/libbddx-0.dll ${prefix}/lib/
+  cp -L ${prefix}/bin/libspotltsmin-0.dll ${prefix}/lib/
+  cp -L ${prefix}/bin/libspotgen-0.dll ${prefix}/lib/
+  rm ${prefix}/bin/libspot-0.dll
+  rm ${prefix}/bin/libspot-0.dll
+  rm ${prefix}/bin/libspotltsmin-0.dll
+  rm ${prefix}/bin/libspotgen-0.dll
```

Comment:
Why?  Libraries should be in `${prefix}/bin`, not ${prefix}/lib`.  Also, _**always**_ use the most specific environment variable, for example `${libdir}` for the directory where the libraries should be, and `${bindir}` for the directory where the executables should be

Reply:
For some reason, these `dll` files ends up in the bin directory after `make install`. In the `lib` directory I only have `.dll.a`. I assumed that I really wanted the `.dll` as I should use a shared and not static library when I build the wrapper in cmake?

I changed that part in the next commit. 

Instead I have two variable `SPOT_LIB_DIR` and `SPOT_INCLUDE_DIR` and I will change the first one in the windows case.

Reply:
> For some reason, these dll files ends up in the bin directory after make install.

The reason is that that's correct.

> In the lib directory I only have .dll.a. I assumed that I really wanted the .dll as I should use a shared and not static library when I build the wrapper in cmake?

As I already said above, `dll.a` are not static libraries but import libraries